ibex_core.core

```diff
@@ -27,7 +27,8 @@ filesets:
       # XXX: Figure out the best way to switch these two implementations
       # dynamically on the target.
 #      - rtl/ibex_register_file_latch.sv # ASIC
-      - rtl/ibex_register_file_ff.sv # FPGA
+#      - rtl/ibex_register_file_ff.sv # FPGA old
+      - rtl/ibex_register_file_fpga.sv # FPGA
```

Comment:
I suggest to change the comment for the FF Regfile to "generic FF-based" and to keep using this one by default.

Reply:
Done. Thank you!